sky/serve/constants.py

```diff
@@ -33,9 +33,20 @@
 # Autoscaler window size in seconds for request per second. We calculate rps by
 # divide the number of requests in last window size by this window size.
 AUTOSCALER_RPS_WINDOW_SIZE_SECONDS = 60
-# Autoscaler scale frequency in seconds. We will try to scale up/down every
-# `scale_frequency`.
-AUTOSCALER_SCALE_FREQUENCY_SECONDS = 20
+# Autoscaler scale decision interval in seconds.
+# We will try to scale up/down every
+# `decision_interval`.
+AUTOSCALER_DEFAULT_DECISION_INTERVAL = 20
+# Autoscaler default upscale delays in seconds.
+# We will upscale only if the target
+# number of instances is larger than
+# the current alive instances for delay amount of time.
+AUTOSCALER_DEFAULT_UPSCALE_DELAY_SECONDS = 300
+# Autoscaler default downscale delays in seconds.
+# We will downscale only if the target
+# number of instances is smaller than the
+# current alive instances for delay amount of time.
+AUTOSCALER_DEFAULT_DOWNSCALE_DELAY_SECONDS = 300
```

Comment:
nit: try to reduce the line of comments as mentioned before. Same for all following constants